src/components/ui/checkbox.tsx

```diff
@@ -0,0 +1,30 @@
+'use client'
+
+import { cn } from '@/lib/utils'
+import * as CheckboxPrimitive from '@radix-ui/react-checkbox'
+import * as React from 'react'
+
+import IconCheck from './icons/IconCheck'
```

Comment:
pls use alias for icons too

Reply:
I will refactor icons usage in the next PR

Reply:
LGTM my bad)